app/src/main/AndroidManifest.xml

```diff
@@ -99,15 +101,5 @@
 
         <!-- Registering support for Amazon Android Push Notifications -->
         <amazon:enable-feature android:name="com.amazon.device.messaging" android:required="false"/>
-        <!-- Warnings about " is not assignable" can be safely ignored. -->
```

Comment:
Do you have a kindle fire tablet to test this on as well?

Reply:
I don't :(. If you can get me one, I am happy to test on it.